interfaces/Portalicious/src/app/pages/project/project-team/add-user-button/add-user-button.component.html

```diff
@@ -1,7 +1,94 @@
 <p-button
-  label="Add user to team"
-  i18n-label
-  rounded
+  label="Add team member"
+  i18n-label="@@add-team-member"
+  outlined
   icon="pi pi-plus"
-  (click)="onAdd()"
+  class="text-xs sm:text-base"
+  (click)="formVisible.set(true)"
 />
+
+<app-form-sidebar
+  [(visible)]="formVisible"
+  [formGroup]="formGroup"
+  [mutation]="assignUserMutation"
+  formTitle="Add team member"
+  i18n-formTitle="@@add-team-member"
+>
+  <app-form-field-wrapper
+    label="Choose user"
+    i18n-label
+    [errorMessage]="formFieldErrors()('userValue')"
+    class="mb-3"
+  >
+    <p-dropdown
+      [options]="unassignedUsers()"
+      [optionLabel]="'username'"
+      [optionValue]="'id'"
+      [filter]="true"
+      formControlName="userValue"
+      placeholder="Type or select"
+      i18n-placeholder
+      class="block w-full"
+      styleClass="w-full"
+      [loading]="allUsers.isPending() || !projectUsers"
+    >
+    </p-dropdown>
+  </app-form-field-wrapper>
+
+  <app-form-field-wrapper
+    label="Assign roles"
+    i18n-label
+    [errorMessage]="formFieldErrors()('rolesValue')"
+    class="mb-3"
+  >
+    <p-multiSelect
+      [options]="roles!.data()"
```

Comment:
I can't imagine the excl being necessary but maybe I'm wrong

```suggestion
      [options]="roles.data()"
```